src/wasm/kotlin/java/arcs/entity-class.kt

```diff
@@ -0,0 +1,117 @@
+// This is a reference for the entity classes generated by schema2wasm, based on the schema:
+//
+// particle Test
+//   data: reads {num: Number, txt: Text, lnk: URL, flg: Boolean, ref: &{val: Text}}
+
+package arcs
+
+class Test_Data_Ref() // Implementation omitted for brevity
```

Comment:
Does this, and the other classes need to use Capital_Snake_Case?

Reply:
Yes. This is generated code; the names are produced from particle/handle/field name combinations and the underscores disambiguate and potentially prevent conflicts.